docs/user/technical-reference/03-10-architecture-updates.md

```diff
@@ -0,0 +1,20 @@
+# Serverless Architecture Updates
+
+## Changes
+
+- The internal Docker Registry is no longer part of the Serverless module. Instead, the Docker Registry is now a separate, standalone module.
+- There is no longer a build job for Functions. Instead, a base image is used, which mounts the required dependencies dynamically.
+- Libraries and dependencies are downloaded at the start of the Function's execution. This means that each replica of the Function can potentially use a different version of the dependencies.
+- Function code is now injected directly into the runtime Pod, eliminating the need for pre-built images.
+
+## Benefits
+
+- **Simplified architecture**: By separating the Docker Registry into its own module, the Serverless module is now more lightweight and easier to manage.
```

Comment:
customers would not care that we have a simple architecture - focus on whats in it for them

Reply:
Done